apps/functions-subscription/src/domain/subscription-history.ts

```diff
@@ -57,6 +64,18 @@ export const makeSubscriptionHistory = (subscription: Subscription) => {
   );
 };
 
+export const updateSubscriptionHistory = (
```

Comment:
**note**: we are not updating the subscription history, but we are creating a new version. maybe we could rename this function accordingly.
what do you think?

Reply:
Yes, what about renaming it to `makeSubscriptionHistoryNextVersion`?

Reply:
Sounds good to me!

Reply:
Done ✅